docs/backup_and_restore.rst

```diff
@@ -160,7 +160,7 @@ backup:
 .. code:: sh
 
    cd install_files/ansible-base
-   ansible-playbook -i inventory -u <SSH username> -K -t backup securedrop-prod.yml -e restore_file="<your backup archive filename>"
+   ansible-playbook -u <SSH username> -K -t backup securedrop-prod.yml -e restore_file="<your backup archive filename>"
```

Comment:
Opened #1789 to discuss. I agree that it's a good idea, but would prefer to keep it out of this PR to aid in review.